### PR TITLE
Add schemas and test demo

### PR DESCRIPTION
**Pull request by u2**
1. add rpc schema
2. add test fixtures demo
3. add data demo

### CODE REVIEW

**Comment by TheWaWaR** [comment]
Looks OK to me.

**Comment by TheWaWaR** [comment]
Missing method? 
* cita_getTransaction
* net_peerCount

Reference: [method.rs](https://github.com/cryptape/cita/blob/develop/jsonrpc_types/src/method.rs#L31-L55)

**Comment by u2** [comment]
@TheWaWaR 

Added.

**Comment by u2** [comment]
@TheWaWaR 

Added.

**Comment by TheWaWaR** [comment]
OK to me now.

File: rpc/schemas/block.json [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+                        },
+                        "transactionsRoot": {
+                            "type": "string",
+                            "description": "DATA, 32 Bytes - the root of the transaction trie of the block."
+                        },
+                        "receiptsRoot": {
+                            "type": "string",
+                            "description": "DATA, 32 Bytes - the root of the receipts trie of the block."
+                        },
+                        "gasUsed": {
```
Comment (kaikai1024) [inline rpc/schemas/block.json]: Still use `gas`?

Comment (u2) [reply rpc/schemas/block.json]: Yes, according to the existing rpc format.

**Pull request merged**